### PR TITLE
Scaled unit ratios and `overrideManifest`

### PR DESCRIPTION
**Pull request by michaelfig**
## Description

Fixes a hardcoded price scaling ratio to be calculated for the actual brands.  Also, to prepare for installing just this change on devnet, provide an `overrideManifest` in the governance proposals to make the manifest explicit in the core proposal.

### Security Considerations

Improved legibility of core proposals generated by writeCoreProposal and extractCoreProposalBundles.

### Documentation Considerations

None.

### Testing Considerations

Tested by run-protocol governance unit tests and chain integration testing, as well as manually inspecting a generated core-proposal.

### CODE REVIEW

**Review by dckc — APPROVED** [review]
Yay! manifest in the proposal!

```js
const manifestInstallRef = "board05669";
const getManifestCall = harden([
  "getManifestForInviteCommittee",
  {
    econCommitteeCharterRef: "board02568",
    voterAddresses: {
      Dan: "agoric1j5sm4kr2mdae6sfey6qjm9vaamddmp0yuxs6kz",
      Michael: "agoric1y5gpzqd876gqmkyl2c5q66vp6e30se4vwn644v",
      Rowland: "agoric14nt7n99jstnk7zzgmw4hzhc05yqjplerrv9uqd",
    },
  },
]);
const overrideManifest = {
  inviteCommitteeMembers: {
    consume: {
      agoricNames: true,
...
```

Calling it an override rather than the normal case doesn't seem ideal, but I can see how it's expedient. Works for me.

File: packages/run-protocol/src/vaultFactory/addAssetToVault.js [hunk 1]
```diff
@@ -87,10 +87,37 @@ export const registerScaledPriceAuthority = async ({
...
 
-  // we used 4 decimals for ibcAtom above...
-  // 1_000_000 atomBrand = 10_000 ibcAtomBrand
-  const scaleIn = makeRatio(100n, atomBrand, 1n, ibcAtomBrand);
-  const scaleOut = makeRatio(1n, usdBrand, 1n, runBrand);
+  // We need "unit amounts" of each brand in order to get the ratios right.  You
+  // can ignore decimalPlaces when adding and subtracting a brand with itself,
+  // but not when creating ratios.
+  const getDecimalP = async brand => {
+    const displayInfo = E(brand).getDisplayInfo();
```
Comment (turadg) [inline packages/run-protocol/src/vaultFactory/addAssetToVault.js:94]: we're veering more from the design that these are merely display info.

if they are truly display info, then couldn't we get this info from somewhere other than the brand? 

if they aren't merely display info, we should revise the design. Not in this PR but if you agree I can make a ticket to follow up.

Comment (dckc) [reply packages/run-protocol/src/vaultFactory/addAssetToVault.js:94]: I agree that it's worth getting wider agreement on whether this justification is sound. A ticket seems reasonable; perhaps we can talk about it this afternoon with Zoe stuff.

Comment (turadg) [reply packages/run-protocol/src/vaultFactory/addAssetToVault.js:94]: https://github.com/Agoric/agoric-sdk/issues/5348

**Pull request merged**